noah/src/main/resources/crawler/pahoconf.conf

```diff
@@ -0,0 +1,9 @@
+[domains]
```

Comment:
Yes. It does.

Done.

I have to add how to run it work in README.md.

It just has one link to crawl every week.

Done.